### DISCUSSION TITLE
Why doesn't the `main` branch contain the `.zshrc` file as a reference for configuration?

### DISCUSSION THREAD

**Discussion by kohane27** [marlonrichert/zsh-autocomplete#629] [Q&A]
Please forgive me for asking a stupid question: Why doesn't the `main` branch contain the `.zshrc` file as a reference for configuration?

I'm currently trying to disable some of the keybindings and disable `recent-dirs`.

Is the file https://github.com/marlonrichert/zsh-autocomplete/blob/5cc9da132e7535a540fb1235ce27fd5a233d4f0e/.zshrc still valid and the latest? It was edited 5 months ago.

Then I also found the thread https://github.com/marlonrichert/zsh-autocomplete/issues/346 which contains https://github.com/marlonrichert/zsh-autocomplete/blob/b41b5635c3a6b11cc14c46e80f32c4c292c63539/.zshrc#L11 which was 2 years ago. So does the line `zstyle ':autocomplete:*' recent-dirs off` still works?

According to https://github.com/marlonrichert/zsh-autocomplete/pull/477, do I have to add the line `zstyle -T ":autocomplete:recent-dirs" disabled` to disable recent-dirs?

Therefore I have the question: Why doesn't the `main` branch contain the `.zshrc` file as a reference for configuration? I could not find the settings contained in the `.zshrc` file in the README.md or in the source code. I could only find them from searching through the issues. But I am afraid I found the old config that no longer works.

Thank you!

**Comment by marlonrichert** [comment]
> Why doesn't the `main` branch contain the `.zshrc` file as a reference for configuration?

Because the currently possible configuration settings are now documented in the Readme.

**Reply by pep-sanwer** [reply]
If possible, it'd be lovely to have a wiki or full listing of config and their possible options listed out for those who require it

Had a similar situation (old configuration) as @kohane27 and took some digging to surface this

Regardless, very appreciative of this incredible tool!
